tests/mentat_prompt_session_test.py

```diff
@@ -0,0 +1,55 @@
+import os
+import subprocess
+from textwrap import dedent
+
+import pytest
+from prompt_toolkit import PromptSession
+
+from mentat.app import run
+
+# Unfortunately these tests just can't be run on Windows
+pytestmark = pytest.mark.skipif(
+    os.name == "nt",
+    reason="PromptSession throws an error on Github Actions Windows runner",
+)
```

Comment:
does it work on Windows though?

Reply:
Yes, I don't see why it shouldn't work on Windows (although I'll definitely test it myself before merging); the only problem is with Github Actions windows runner (which doesn't like prompttoolkit for whatever reason.

Reply:
Just confirmed; it does work on windows